src/main/java/frc/robot/RobotContainer.java

```diff
@@ -144,15 +179,36 @@ private void configureBindings() {
 
     fieldCentricButton.onTrue(swerveDrive.toggleFieldCentric());
 
-    resetFieldCentricButton.onTrue(new InstantCommand((
-    ) -> {
+    resetFieldCentricButton.onTrue(new InstantCommand(() -> {
       swerveDrive.resetFieldCentric();
     }, swerveDrive));
 
-    turtleButton.whileTrue(new RunCommand((
-    ) -> {
+    turtleButton.whileTrue(new RunCommand(() -> {
       swerveDrive.turtle();
-    }, swerveDrive));  }
+    }, swerveDrive));
+
+    intakeButton.whileTrue(new RunCommand(() -> {
+      intakeSubsystem.intake(limitX.calculate((applyDeadband(-leftJoystick.getThrottle(), IntakeConstants.INTAKE_DEADBAND))));
+    }, intakeSubsystem));
+
+    feedInButton.whileTrue(new RunCommand((
+    ) -> {
+      shooter.feed(0.1);
+    }, shooter));
+    feedInButton.onFalse(new InstantCommand((
+    ) -> {
+      shooter.feed(0);
```

Comment:
I recommend making a default command of the feeder subsystem that sets the motor speed to 0